test/cmd/newapp.sh

```diff
@@ -97,7 +97,7 @@ os::cmd::expect_success 'oc delete all -l app=ruby'
 
 # check new-build
 os::cmd::expect_failure_and_text 'oc new-build mysql -o yaml' 'you must specify at least one source repository URL'
-os::cmd::expect_success_and_text 'oc new-build mysql --binary -o yaml' 'type: Binary'
+os::cmd::expect_success_and_text 'oc new-build mysql --binary -o yaml --to mysql:bin' 'type: Binary'
```

Comment:
Hrm - this makes obvious a real problem with the current new-build flow - I think that I want mysql not to default to latest on the import in this case.  I think I would expect the outcome to be:

1. Create a new image stream mysql
2. Set a tag in the image stream to the input location, "base" (maybe?) and import only that tag (has backward compatibility implications)
3. Set the output tag to latest.